src/rail/estimation/algos/deepdisc.py

```diff
@@ -10,15 +37,190 @@ def __init__(self, args, comm=None):
         CatInformer.__init__(self, args, comm=comm)
 
     def run(self):
-        pass
+        """
+        Train a inception NN on a fraction of the training data
+        """
+        # Not sure what .get_data('input') is ?
+        train_data = self.get_data('input')
+        
+        cfgfile = self.config.cfgfile
```

Comment:
Typically, what I've seen for these `self.config.<thing>` values is a default and a help message defined in a section just after line 7/34 `config_options = CatInformer.config_otions.copy()`.

They typically look like this: 
```
cfgfile = Param(str, None, required=True,
            msg="The primary configuration file for the deepdisc models."),
batch_size = Param(int, 10_000, required=False,
            msg="The batch size ..."),
```

This isn't something that we need to establish right now, but ultimately it will make rail_deepdisc much more user friendly because users will be able to type something like `help(DeepDiscInformer)` and will get a nicely formatted help message that includes all the allowable input.

Reply:
Good to keep in mind.  I'll add a comment there to include that later